nvflare/app_common/filters/convert_weights.py

```diff
@@ -21,14 +21,18 @@
 
 class ConvertWeights(Filter):
 
-    """
-    This filter converts from
-    """
-
     WEIGHTS_TO_DIFF = "weights_to_diff"
     DIFF_TO_WEIGHTS = "diff_to_weights"
 
     def __init__(self, direction: str):
+        """Convert WEIGHTS to WEIGHT_DIFF or vice versa
+
+        Args:
+            direction (str): control conversion direction.  Either weights_to_diff or diff_to_weights.
+
+        Raises:
+            ValueError: [description]
```

Comment:
should this [description] be changed to something else?

Reply:
Added the description of condition when ValueError is raised.